src/app/models/constraint-models/combination-constraint.ts

```diff
@@ -31,7 +30,7 @@ export class CombinationConstraint extends Constraint {
       });
     }
     this.combinationState = state ? state : CombinationState.And;
-    this.mark = mark ? mark : ConstraintMark.OBSERVATION;
+    this.dimension = dimension ? dimension : Constraint.TOP_LEVEL_DIMENSION;
```

Comment:
`this.dimension = dimension ? dimension : CombinationDimension.SUBJECT;`

also see related comments below